app/src/main/java/com/eveningoutpost/dexdrip/Models/JoH.java

```diff
@@ -190,6 +191,18 @@ public static long msSince(long when) {
         return (tsl() - when);
     }
 
+    public static long msSince(long end, long start) {
+        return (end - start);
+    }
+
+    public static String toHumanTime(double t) {
+        return toHumanTime((long)t);
+    }
+
+    public static String toHumanTime(long t) {
```

Comment:
Does `dateTimeText` not meet your needs? Also is this method even referenced by the PR?

Reply:
This was used in debugging this PR. In any case I'll use dateTimeText.